cmd/snap-confine/snap-confine-invocation.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
```

Comment:
this is so fundamental/passed around that it might as well be "snap-confine.h"

Reply:
Done now.

Reply:
I'm confused, I thought this comment was moving `sc_invocation` into `snap-confine.h` - am I misunderstanding the comment? I mean, its marked as "Done now" and "resolved" and yet we still have `sc_invocation` in snap-confine-invocation.h - what am I missing :)

Reply:
This was reverted as indicated here https://github.com/snapcore/snapd/pull/6575#issuecomment-473858513